spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -190,9 +222,24 @@ private Map setupPySparkEnv() throws IOException{
       }
     }
 
+    LOGGER.debug("PYTHONPATH: " + env.get("PYTHONPATH"));
     return env;
   }
 
+  // Run python shell
+  // Choose python in the order of
+  // PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python
```

Comment:
This is done in ZEPPELIN-2195.  The order should be `PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python`. `zeppelin.pyspark.python` only affect the driver side, won't affect the executor side. So we should use the spark standard configuration. IMO, zeppelin should use spark standard configuration as possible as it can, rather than creating new configuration.